WooCommerce/WooCommerceTests/Authentication/AuthenticationManagerTests.swift

```diff
@@ -496,6 +512,44 @@ final class AuthenticationManagerTests: XCTestCase {
         XCTAssertTrue(try XCTUnwrap(analyticsProvider.receivedProperties.first?["is_jetpack_connected"] as? Bool))
         XCTAssertEqual(analyticsProvider.receivedProperties.first?["url_after_redirects"] as? String, siteInfo.url)
     }
+
+    func test_it_presents_store_creation_flow_when_there_are_no_valid_stores() throws {
+        // Given
+        let featureFlagService = MockFeatureFlagService(isStoreCreationM2Enabled: true,
+                                                        isStoreCreationM2WithInAppPurchasesEnabled: true)
```

Comment:
nit: any reasons this IAP feature flag needs to be `true`? it's currently disabled without a concrete plan to launch it

Reply:
Removed this feature flag mocking in https://github.com/woocommerce/woocommerce-ios/pull/9444/commits/53d14e7d4d9d4f399300f71b26bf1acfc6d5dd9f